src/App.js

```diff
@@ -38,6 +38,12 @@ function App() {
   };
 
   const handleGenerateClick = (numOfWeek) => {
+    if (WEEKLY_TASKS.length !== MEMBERS.length) {
+      throw new Error(
+        ` ${WEEKLY_TASKS} and ${MEMBERS} array length are not matching to each other`
```

Comment:
```WEEKLY_TASKS``` and ```MEMBERS``` both are arrays and the content in the array should not be in the error message. We can just do ```WEEKLY_TASKS and MEMBERS array length are not matching to each other```, but with string literal instead of "".